python/lsst/ts/envsensors/device/rpi_serial_hat.py

```diff
@@ -48,6 +53,50 @@ class RpiSerialHat(BaseDevice):
     my hands on a raspberry pi with a serial hat.
     """
 
+    # Define serial hub unit port identifiers
+    SERIAL_CH_0: str = "serial_ch_0"
+    SERIAL_CH_1: str = "serial_ch_1"
+    SERIAL_CH_2: str = "serial_ch_2"
+    SERIAL_CH_3: str = "serial_ch_3"
+    SERIAL_CH_4: str = "serial_ch_4"
```

Comment:
Type annotations aren't needed for constants and they don't usually add clarity.

(Also they often aren't needed when you set a variable to the value returned by a function, though sometimes those do add clarity, and they are needed if the value can have multiple types, not just the type returned by the first function call.)

Reply:
Legacy code from a previous developer and I didn't like it either but didn't want to touch the code too much since it is rather complex and actually too complex so it does deserve to be cleaned up. I removed these type annotations.